data/json/recipes/recipe_others.json

```diff
@@ -6734,6 +6734,31 @@
       ["fur", 3]
     ]
   ]
+},{
+    "type" : "recipe",
+    "result": "primitive_adze",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "skills_required": [ "survival", 2 ],
+    "difficulty": 2,
+    "time": 60000,
+    "reversible": false,
+    "autolearn": true,
+    "qualities":[
+      {"id":"HAMMER","level":2,"amount":1},
+      {"id":"CUT","level":1,"amount":1}
+    ],
+    "components": [
+      [ [ "stick", 1 ], 
+        [ "2x4", 1 ] ],
+      [ [ "rock", 1 ],
+        [ "ceramic_shard", 1 ],
+        [ "sharp_rock", 1 ] ],
```

Comment:
Syntax issue: easier to read these if the brackets that open and close a group are on their own lines.  Not a blocker of itself, though, and I'll take care of it in the merge.